plugins/scaffolder-backend/package.json

```diff
@@ -44,7 +44,8 @@
     "nodegit": "0.27.0",
     "uuid": "^8.2.0",
     "winston": "^3.2.1",
-    "yaml": "^1.10.0"
+    "yaml": "^1.10.0",
+    "node-fetch": "^2.6.1"
```

Comment:
Use `cross-fetch`, right @benjdlambert? :D

Reply:
Yes! Now the #2831 is merged!